.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  parserOptions: {
+    ecmaVersion: 6,
+  },
+  env: {
+    node: true,
+    mocha: true,
+  },
+  'extends': 'eslint:recommended',
```

Comment:
You could use the Google JS ESLint config: https://github.com/google/eslint-config-google, maybe [this config](https://github.com/google/eslint-config-google#using-the-google-config-with-eslintrecommended).

Reply:
Ah great, you have one ready :)